Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -12,11 +12,14 @@ private func getDate(offset: Int, from: NSDate = NSDate()) -> NSDate {
     return calendar.dateFromComponents(nowComponents)!
 }
 
-class HistoryPanel: SiteTableViewController {
-    private let NumSections = 4
-    private var Today = getDate(0)
-    private var Yesterday = getDate(-1)
-    private var ThisWeek = getDate(-7)
+private let NumSections = 4
```

Comment:
Defining it outside the class makes it "static" since it's only defined once for all instances of the class. Isn't that how we've been defining all of our "static" vars? [\[1\]](https://github.com/mozilla/firefox-ios/blob/master/Client/Frontend/Browser/BrowserViewController.swift#L11)[\[2\]](https://github.com/mozilla/firefox-ios/blob/master/Client/Frontend/Home/HomePanelViewController.swift#L9)[\[3\]](https://github.com/mozilla/firefox-ios/blob/master/Client/Frontend/Browser/URLBarView.swift#L230), etc. I think you implemented that third one -- what's different in that case?

Don't feel strongly about it though, though, so I'll revert this. I assume we'll be changing these all to proper statics when Swift supports them.